source/val/validate_instruction.cpp

```diff
@@ -449,6 +449,17 @@ spv_result_t InstructionPass(ValidationState_t& _, const Instruction* inst) {
     CheckIfKnownExtension(_, inst);
   } else if (opcode == SpvOpCapability) {
     _.RegisterCapability(inst->GetOperandAs<SpvCapability>(0));
+  } else if (opcode == SpvOpExtInstImport) {
```

Comment:
I would prefer this get into a new file, validate_extensions.cpp, and get invoked as ExtensionsPass between annotation and mode checks.

Reply:
There is a validate_ext_inst.cpp that implements ExtInstPass. Should I add this code there instead of adding a new file and pass function?

Reply:
Though on second look, it looks more like it is dealing with the specific calls into extension as opposed to the import.

Reply:
Done